puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/packaging_build_rpm.groovy

```diff
@@ -0,0 +1,47 @@
+pipeline {
+    agent { label 'rpmbuild' }
+
+    options {
+        timestamps()
+        timeout(time: 2, unit: 'HOURS')
+        ansiColor('xterm')
+    }
+
+    stages {
+        stage("Setup Environment") {
+            steps {
+                git(url: 'https://github.com/theforeman/foreman-packaging/', branch: env.branch)
+                setup_obal()
+            }
+        }
+
+        stage("Build RPM") {
+            steps {
+                script {
+                    sh("mkdir -p ${env.WORKSPACE}/rel-eng/build")
+
+                    def build_package_tito_releaser_args = "-o ${env.WORKSPACE}/rel-eng/build/"
+
+                    if (env.tag) {
```

Comment:
I wonder if we use this anywhere. I don't see it in jobs and we don't use `tito tag` anymore so I think you can remove it.

Reply:
perfect. more things to delete!